src/main/java/me/daxanius/npe/mixins/server/ServerMetadataMixin.java

```diff
@@ -0,0 +1,12 @@
+package me.daxanius.npe.mixins.server;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import me.daxanius.npe.config.NoPryingEyesConfig;
+import net.minecraft.server.ServerMetadata;
+
+@Mixin(ServerMetadata.class)
+public class ServerMetadataMixin {
+    @SuppressWarnings("unused")
+    private boolean preventsChatReports = NoPryingEyesConfig.getInstance().noKey();
```

Comment:
This probably does not work because you're not doing anything with this new variable. I'm personally against suppressing warnings as it may hide glaring issues that are otherwise unknown.

According to the [NCR page on this](https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/How-to-Get-Safe-Server-Status), you should add this as a property for the serialized ServerStatus class, which gets sent to the client. This line of code is just sitting here without doing anything.

Reply:
ServerStatus  == ServerMetadata
just that former is mojmap, latter is yarn

Will try to annotate with unique and make it public though (private works for NCR for some reason though)

Reply:
Did try unique and public, no effect
Also tried removing the `abstract` keyword from the class declaration just in case, no effect (as expected)

Maybe I should try it outside dev env?

Reply:
Fixed